panel/io/fastapi.py

```diff
@@ -0,0 +1,319 @@
+from __future__ import annotations
+
+import asyncio
+import uuid
+
+from functools import wraps
+from typing import TYPE_CHECKING, Mapping, cast
+
+from bokeh_fastapi import BokehFastAPI
+from bokeh_fastapi.handler import WSHandler
```

Comment:
Do you want to add a proper error message here if the import fails given that this is not a required dependency?

Reply:
I have added an error message, and I'm open to suggestions. 

Chose `bokeh_fastapi` with an underscore because this is what other bokeh conda projects have [[ref](https://anaconda.org/bokeh/)], and conda will not install `bokeh-fastapi` out of the box.  Pip is smarter and will install no matter if you install it as `bokeh_fastapi` or `bokeh-fastapi`.